src/types/requests.ts

```diff
@@ -32,6 +32,15 @@ export type PricesRequest = {
 
 export type MarketRequest = Nft
 
+export type AuctionRequest = {
+  loanId: Address
```

Comment:
load id is always required? just double-checking because some of the other actions have it optional

Reply:
for auction/bid and auction/claim are mandatory